proteome-rest/src/main/resources/application.properties

```diff
@@ -34,7 +34,7 @@ spring.data.solr.connectionTimeout=20000
 spring.data.solr.socketTimeout=3600000
 
 # Solr properties
-solr.collectionName=genecentric
+solr.collectionName=proteome
```

Comment:
Do you know where do we use this? because you changed from genecentric to proteome, have you noticed anything?

Reply:
Nope, i didnt notice any issue because of this change.